packages/css/src/badge.css

```
@@ -1,91 +1,93 @@
 .ds-badge {
-  /* color vars */
   --dsc-badge-background: var(--ds-color-base-default);
   --dsc-badge-color: var(--ds-color-contrast-default);
-
-  /* What do we do here? */
-
-  &[data-color='neutral'] {
-    --dsc-badge-background: var(--ds-color-neutral-surface-default);
-    --dsc-badge-color: var(--ds-color-neutral-text-default);
-  }
-
-  /* sizing vars */
   --dsc-badge-padding: 0 calc(var(--ds-spacing-1) + calc(var(--ds-spacing-1) / 2));
   --dsc-badge-size: calc(var(--ds-sizing-3) + calc(var(--ds-spacing-1) / 2));
 
-  box-sizing: border-box;
-  display: inline-flex;
-  line-height: var(--ds-line-height-sm);
-  position: relative;
-
   &::before {
     content: attr(data-count);
     background: var(--dsc-badge-background);
     border-radius: var(--ds-border-radius-full);
     box-sizing: border-box;
     color: var(--dsc-badge-color);
-    display: grid;
+    display: inline-grid;
     font-size: var(--ds-font-size-minus-1); /* Default to small when size is not defined */
     min-height: var(--dsc-badge-size);
     min-width: var(--dsc-badge-size);
     padding: var(--dsc-badge-padding);
     place-items: center;
+    box-sizing: border-box;
+    line-height: var(--ds-line-height-sm);
+    width: fit-content;
   }
 
-  & :where(img, svg) {
-    flex-shrink: 0; /* Never shrink icon */
-    font-size: 1.25em; /* Auto scale icon based on font-size */
+  /* What do we do here? */
+
+  &[data-color='neutral'] {
+    --dsc-badge-background: var(--ds-color-neutral-surface-default);
+    --dsc-badge-color: var(--ds-color-neutral-text-default);
   }
+}
 
-  /* If placement is present, we are floating */
-  &[data-placement]::before {
+.ds-badge--placement {
```

Should this be renamed as well? ☺️ 
```suggestion
.ds-badge--position {
```
and, do we use `.ds-badge--position` or `.ds-badge-position`? 🤔

---

hm yea this should be renamed. I think double dashes since it is an extension of `.ds-badge` 🤔